src/sentry/sentry_metrics/indexer/ratelimiters.py

```diff
@@ -11,67 +11,65 @@
     RequestedQuota,
     Timestamp,
 )
-from sentry.sentry_metrics.configuration import UseCaseKey, get_ingest_config
+from sentry.sentry_metrics.configuration import RateLimiterNamespace, get_writes_ratelimiter_config
 from sentry.sentry_metrics.indexer.base import FetchType, FetchTypeExt, KeyCollection, KeyResult
 from sentry.utils import metrics
 
 OrgId = int
 
 
-def _build_quota_key(use_case_id: UseCaseKey, org_id: Optional[OrgId] = None) -> str:
-    use_case_str = {
-        UseCaseKey.PERFORMANCE: "performance",
-        UseCaseKey.RELEASE_HEALTH: "releasehealth",
-    }[use_case_id]
+def _build_quota_key(namespace: RateLimiterNamespace, org_id: Optional[OrgId] = None) -> str:
 
     if org_id is not None:
-        return f"metrics-indexer-{use_case_str}-org-{org_id}"
+        return f"metrics-indexer-{namespace.value}-org-{org_id}"
     else:
-        return f"metrics-indexer-{use_case_str}-global"
+        return f"metrics-indexer-{namespace.value}-global"
 
 
 @metrics.wraps("sentry_metrics.indexer.construct_quotas")
-def _construct_quotas(use_case_id: UseCaseKey) -> Sequence[Quota]:
+def _construct_quotas(namespace: RateLimiterNamespace) -> Sequence[Quota]:
     """
     Construct write limit's quotas based on current sentry options.
 
     This value can potentially cached globally as long as it is invalidated
     when sentry.options are.
     """
-    if use_case_id == UseCaseKey.PERFORMANCE:
+    if namespace == RateLimiterNamespace.PERFORMANCE:
```

Comment:
I don't think this is a safe change. The quota applies to the use case not the namespace. Shouldn't the same quote apply for all performance backends?